components/Layout.js

```diff
@@ -73,11 +75,13 @@ export default function Layout(props) {
                   ? '/en/my-dashboard'
                   : '/fr/mon-tableau-de-bord'
               }`,
+              component: Link,
```

Comment:
This doesn't seem to do anything, maybe have a conversation with the DS team to either get an idea on how we should be using this prop or to get them to adjust their component